documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -0,0 +1,567 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Lost track of the naming convention decided on in the emails. Are dashes used for the target, language pair since it's the "language" specification and underscore to separate library?